index.js

```diff
@@ -119,16 +100,81 @@
   //  $$type :: String
   var $$type = '@@type';
 
-  //  type :: Any -> String
+  //  RPARSE :: RegExp
+  var RPARSE = /^([^]+)\/([^]+?)(?:@(\d+))?$/;
```

Comment:
Isn't `[^]` and `.` same-ish? I think `.` is expressing what `[^]` is trying to do, much better.